prow/jobs/kyma/components/remote-environment-broker/remote-environment-broker.yaml

```diff
@@ -0,0 +1,49 @@
+job_template: &job_template
+  optional: true
```

Comment:
optional is used by Tide - currently we don't use this component, please remove it, and remove in place where you copied. If this was mentioned in the documentation, please adjust documentation.

Reply:
Ok, I deleted the `optional` parameter from this file, binding-usage-controller.yaml and from the documentation.